### ISSUE TITLE
how to run on real device? not the emulator

### ISSUE THREAD

**Issue by heyanyanchina123** [chentsulin/react-native-counter-ios-android#1]

**Comment by chentsulin** [comment]
This is not a react-native relative issue, and more relative to iOS and Android their self. 

Official guides as reference:
- [iOS](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/LaunchingYourApponDevices/LaunchingYourApponDevices.html#//apple_ref/doc/uid/TP40012582-CH27-SW4)
- [Android](http://developer.android.com/tools/device.html)

**Comment by heyanyanchina123** [comment]
I mean that how to let Android run on a real device which has a different IP address with the server, on iOS,  NSString *urlString = @"http://localhost:8081/index.ios.bundle?dev=false&minify=true";but on Android how to set the IP address

**Comment by chentsulin** [comment]
Maybe this will help:
https://facebook.github.io/react-native/docs/running-on-device-android.html#content

**Comment by heyanyanchina123** [comment]
@chentsulin ,yeah ,I have read it before, but it can't write the IP address to the Android code directly

**Comment by easyli** [comment]
Fixed on React Native 0.40.0
